src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -268,6 +311,27 @@ class ZHAConfigDashboard extends LitElement {
     this._networkSettings = await fetchZHANetworkSettings(this.hass!);
   }
 
+  private async _fetchDevicesAndUpdateStatus(): Promise<void> {
+    try {
+      const devices = await fetchDevices(this.hass!);
+      const total = devices.length;
+      const online = devices.filter((d) => d.available).length;
+      const offline = total - online;
+      this._totalDevices = total;
+      this._offlineDevices = offline;
+    } catch (err: any) {
+      this._totalDevices = 0;
+      this._offlineDevices = 0;
+      showAlertDialog(this, {
+        title: this.hass.localize(
+          "ui.panel.config.zha.configuration_page.fetch_devices_error_title"
+        ),
+        text: err.message,
+        warning: true,
+      });
```

Comment:
Please use `ha-alert` as I suggested. 

```suggestion
      this._error = err.message || error;
```